core/iwasm/common/wasm_runtime_common.c

```diff
@@ -4847,6 +4881,47 @@ wasm_runtime_invoke_native(WASMExecEnv *exec_env, void *func_ptr,
                 break;
             }
             case VALUE_TYPE_I64:
+#if WASM_ENABLE_MEMORY64 != 0
```

Comment:
Add macro control for `case VALUE_TYPE_I32`

Reply:
And there are two other `wasm_runtime_invoke_native` for other targets, should also modify them.

Reply:
The other two are for 32-bit platform, so I left them unmodified:
```C
#if defined(BUILD_TARGET_ARM_VFP) || defined(BUILD_TARGET_THUMB_VFP) \
    || defined(BUILD_TARGET_RISCV32_ILP32D)                          \
    || defined(BUILD_TARGET_RISCV32_ILP32) || defined(BUILD_TARGET_ARC)
...

#if defined(BUILD_TARGET_X86_32) || defined(BUILD_TARGET_ARM)    \
    || defined(BUILD_TARGET_THUMB) || defined(BUILD_TARGET_MIPS) \
    || defined(BUILD_TARGET_XTENSA)
...
```

Reply:
Got it, thanks